packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -151,6 +151,8 @@ export const messages = {
   isv_debug_bootstrap_generate_launchjson: 'Creating launch configuration',
   isv_debug_bootstrap_open_project:
     'Opening project in new Visual Studio Code window',
+  isv_debug_config_environment_error:
+    'There was a problem configuring the environment for ISV Customer Debugger. Some features may not work. See VS Code Developer Tools and Salesforce CLI logs for details.',
```

Comment:
![image](https://user-images.githubusercontent.com/7286437/41616175-2517ddee-73b2-11e8-885f-bdda6f1794dc.png)

Reply:
It seems that the check that we do is only `if (sfdxApexDebuggerExtension && sfdxApexDebuggerExtension.id)`. So the user might **not** even be using the ISV Debugger and this message could happen.
Is there a way we can re-word this differently?

Reply:
Maybe something like this?
`'Salesforce Extensions for VS Code encountered a problem while configuring your environment. Some features might not work. For details, select Salesforce CLI in the Output panel or click Help > Toggle Developer Tools.'`

Reply:
Yes, that looks good. @DatGuyJonathan could you ensure that the logging has enough details to identify this at the ISV Debugger culprit code path.

A tech debt that we need to file is to provide a better way to dump logs. Just like what VS LIve has with the Dump Logs command.

Reply:
I guess so. We try to avoid mentioning the hidden .sfdx folder, since it's not really designed to be messed with, but it sounds like the best solution here. How about this?

`'Salesforce Extensions for VS Code encountered a problem while configuring your environment. Some features might not work. For details, click Help > Toggle Developer Tools or check the Salesforce CLI logs in ~/.sfdx/sfdx.log.'`

Reply:
Looks good. Thanks @ruthemmanuelle